packages/client/hmi-client/src/components/drilldown/tera-drilldown-section.vue

```diff
@@ -48,6 +48,10 @@ header > div {
 	display: inline-flex;
 	gap: var(--gap-1);
 	align-items: center;
+
+	&:first-child {
+		flex: 1;
+	}
```

Comment:
May be little heavy-handed? There are a lot of component running with tera-drilldown-section and are are changing the first-child without really knowing what the first child is?

Reply:
This is the first child of the header which typically contains a title or instructions. The second child contains buttons. 

I don't think there would be a case where the first child would inhibit the second one. Granting the first child with a `flex: 1` just enables it's children to use `flex: 1`.

Reply:
okay